doc/source/examples.rst

```diff
@@ -77,6 +77,17 @@ To start a service which requires systemd, configure `molecule.yml` with a
 systemd compliant image, capabilities, volumes, and command as follows.
 
 .. code-block:: bash
+			vms.vm.provision :ansible_local do |ansible|
+				ansible.verbose = "v"
+				ansible.install_mode = "pip"
+				ansible.version = "2.4.2.0"
+				#ansible.version = "2.2.1.0"
+				ansible.playbook = "#{PROJECT_NAME}/tests/test.yml"
+				ansible.galaxy_role_file = "#{PROJECT_NAME}/requirements.yml"
+				ansible.galaxy_roles_path = "#{PROJECT_NAME}/tests/.roles"
+				ansible.galaxy_command = "ansible-galaxy install --ignore-certs --role-file=%{role_file} --roles-path=%{roles_path} #{ANSIBLE_GALAXY_FORCE}"
+				ansible.become = true
+			end
```

Comment:
hi @mtbvang what exactly is the above for?  I'm kinda lost with this info above.

Reply:
@retr0h I don't know what happened there. I'll fix it in the PR that includes the comment below too.